src/main/java/backend/UpdateController.java

```diff
@@ -144,4 +145,22 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         // No sort or updated, return sort by descending ID, which is the default.
         return Qualifier.getSortComparator(models, "id", true, false);
     }
+
+    /**
+     * Applies the count qualifier to the set of issues obtained after the sorting order is applied.
+     *
+     * @param issueList Issue list obtained after filtering and sorting.
+     * @param filterExpr The filter expression of the particular panel.
+     * @return The maximum count of issues
+     */
+    private int determineMaxCount(List<TurboIssue> issueList, FilterExpression filterExpr){
+        for (Qualifier metaQualifier : filterExpr.find(Qualifier::isMetaQualifier)) {
+            if (metaQualifier.getType() == QualifierType.COUNT) {
+                int countQualifierNum = metaQualifier.getNumber().get();
+                return countQualifierNum < issueList.size() && countQualifierNum >= 0 ?
```

Comment:
`limit` also handles this comparison for us. You can actually just return `countQualifierNum` here. The 'max' can be removed from the method name to reflect that.